src/styles/blocks/header.scss

```diff
@@ -0,0 +1,141 @@
+.header {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+
+  background-image: url(../images/Header.png);
+  background-repeat: no-repeat;
+  background-position: top;
+  background-size: cover;
+
+  &__content-wrapper {
+    padding: 36px 10%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100%;
+    width: 100vw;
+    max-width: $maxContentWidth;
```

Comment:
it should be 1260 px

Reply:
Hen 1260px max-width is applied, the gaps on the grid are wider than the supposed (24px).